### PR TITLE
Add common conditions

### PR DESCRIPTION
**Pull request by Shatur**
Mirrors https://github.com/bevyengine/bevy/blob/main/crates/bevy_input/src/common_conditions.rs

### CODE REVIEW

File: src/common_conditions.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//! Run conditions for actions.
+
+use crate::{prelude::ActionState, Actionlike};
+use bevy::prelude::Res;
+
+/// Stateful run condition that can be toggled via an action press using [`ActionState::just_pressed`].
+pub fn action_toggle_active<T>(default: bool, action: T) -> impl FnMut(Res<ActionState<T>>) -> bool
+where
+    T: Actionlike + Copy,
```
Comment (alice-i-cecile) [inline src/common_conditions.rs:9]: I'd like to relax these bounds to Clone: there's no reason to enforce Copy here.

File: RELEASES.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Unreleased
+
+### Usability
+
+- Added common conditions for actions that mirrors input conditions in Bevy.
```
Comment (alice-i-cecile) [inline RELEASES.md:7]:
```suggestion
- Added common run conditions for actions that mirrors input conditions in Bevy.

```

File: src/common_conditions.rs (hunk 1 above)
Comment (Shatur) [reply src/common_conditions.rs:9]: But we will clone it every usage because `pressed` and similar take by value. Is it okay?

Comment (alice-i-cecile) [reply src/common_conditions.rs:9]: Not any worse than the `Copy` bound :) In either case, it's getting copied every frame.

**Comment by Shatur** [comment]
Looks like CI fails, but the error is unrelated to the PR.

**Review by alice-i-cecile — APPROVED** [review]

**Comment by Shatur** [comment]
Right, I thought that markdown lint failed because of this:
```
 2023-03-20 23:16:49 [ERROR]   Failed to get [GITHUB_TOKEN]!
2023-03-20 23:16:49 [ERROR]   []
2023-03-20 23:16:49 [ERROR]   Please set a [GITHUB_TOKEN] from the main workflow environment to take advantage of multiple status reports!
```
But the actual error was this:
```
2023-03-10 21:40:29 [ERROR]   Found errors in [markdownlint] linter!
2023-03-10 21:40:29 [ERROR]   Error code: 1. Command output:
------
/github/workspace/RELEASES.md:9 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
------
```

**Pull request merged**